vagrant/aws/aws-init.sh

```diff
@@ -18,16 +18,31 @@
 # or bring up your mini Kafka cluster.
 
 # Install dependencies
-sudo apt-get install -y maven openjdk-6-jdk build-essential \
-            ruby-dev zlib1g-dev realpath python-setuptools
+sudo apt-get install -y \
+  vagrant \
+  maven \
+  openjdk-8-jdk \
```

Comment:
Also, just noticed this, but we might want to put openjdk-8-jdk-headless instead. It's about half the size of the non-headless version.